.travis.yml

```diff
@@ -13,22 +13,16 @@ env:
         - SYMFONY_PHPUNIT_REMOVE="symfony/yaml" # keep Prophecy
 
 php:
-    - 5.6
-    - 7.0
     - 7.1
     - 7.2
     - 7.3
     - nightly
 
 matrix:
     include:
-        - php: 5.6
-          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.1
```

Comment:
I guess that, now that 7.1 is the minimum version, we should move old 5.6 options here

Reply:
yep, one build should be with `--prefer-lowest` to detect when new code introduces incompatibilities with old versions (so that its either fixed or minimum requirements are increased)

Reply:
Ok so would you add another matrix entry leaving also `dev` dependencies and `SYMFONY_VERSION` (basically having three entries with PHP7.1 as version)?

Reply:
Moreover should I add an `allow_failures` with the nightly build? I think we should as PHP is moving faster than this bundle does.

Reply:
afaik nightly is now php 8 which makes no sense to test as its not supported by anything yet... i'd just remove that from the matrix.

the 3 entries with php 7.1 all do different things. though i guess the idea of the one with `dev` was to use the newest supported php version, so it could be changed to 7.3.

Reply:
Maybe 7.4snapshot makes sense?

Reply:
yeah that would make sense - could be allowed to fail maybe, as its not necessarily working correctly.

Reply:
Fix pushed. Does it LGTY?
Moreover with PHP7.3 and dev deps we found a potential issue with (I suppose) `phpunit-bridge` --> https://travis-ci.org/DonCallisto/KnpMenuBundle/jobs/548106697 but is not related with this issue. If everything is good with this PR I can also fix that problem in a different PR.